tests/end-to-end.rs

```diff
@@ -83,6 +83,7 @@ test_case!(batch, "BatchStack", &["golang", "java"]); //java fails cdk synth bc
 test_case!(cloudwatch, "CloudwatchStack", &["golang"]);
 test_case!(ecs, "EcsStack", &["java", "golang"]);
 test_case!(ec2, "Ec2Stack", &["java", "golang"]);
+test_case!(efs, "EfsStack", &["golang"]);
```

Comment:
You've got a failing java test. There are intermittent failures due to certain data structures in Java being non deterministic so add `"java"` here and your tests should pass.